pyaptamer/datasets/_loaders/_li2014.py

```diff
@@ -0,0 +1,55 @@
+__author__ = "satvshr"
+__all__ = ["load_train_li2014", "load_test_li2014"]
+import os
+
+import pandas as pd
+
+
+def load_train_li2014():
+    """
+    Load the Li 2014 training dataset.
+
```

Comment:
please add some description of the dataset here. What are columns, what do they mean, what can the values be, etc.